topics/metagenomics/tutorials/plasmid-metagenomics-nanopore/tutorial.md

```diff
@@ -0,0 +1,390 @@
+>
+> To obtain high-quality short read data, paired-end (2 × 150 bp) libraries were prepared by the focused acoustic shearing method with the NEBNext Ultra DNA Library Prep Kit and the Multiplex Oligos Kit for Illumina (NEB). The libraries were quantified by employing quantitative PCR with P5-P7 primers, and they were pooled together and sequenced on the NextSeq 500 platform according to the manufacturer's protocol (Illumina).
+>
+> Although a local basecaller script was used during the run, there was still a small amount of reads that were not basecalled due to the generation of raw data in a rapid mode. Albacore basecalling software (v1.0.3) was used to generate fast5 files harboring the 1D DNA sequence from fast5 files with only raw data in the tmp folder. Also, the read_fast5_basecaller.py script in Albacore was used to de-multiplex the 12 samples from basecalled fast5 files (except the files in fail folder) based on the 12 barcodes in SQK-RBK001. The Poretools toolkit was utilized to extract all the DNA sequences from fast5 to fasta format among the 12 samples, respectively (Poretools, RRID:SCR_015879).
+>
+{: .details}
+
+To make this tutorial easier to execute, we are providing only one MDR pasmid-bearing strain.
+
+> ### {% icon comment %} Dataset details
+> Because of the large size of the original datasets (1.15 GB) you are given 1 of the 12 plasmids
+> files.
+> <br><br>
+> This sequence file is 51 MB of nanopore sequences. The 10026 reads found in this file contain 49190798 nucleotides.
+> As mentioned before nanopore sequences are long reads and this is confirmed by a mean read length of 4906.3.
+{: .comment}
+
+## Importing the data into Galaxy
+
+Now that we know what our input data is, let's get it into our Galaxy history:
+
+> ### {% icon hands_on %} Hands-on: Obtaining our data
+>
+> 1. Make sure you have an empty analysis history. Give it a name.
+>
+>    > ### {% icon tip %} Starting a new history
+>    >
+>    > * Click the **gear icon** at the top of the history panel
+>    > * Select the option **Create New** from the menu
+>    {: .tip}
+>
+> 2. **Import Sample Data** [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.1443246.svg)](https://doi.org/10.5281/zenodo.1443246)
+>    ```
+>    https://zenodo.org/record/1443246/files/RB01.fasta
+>    ```
+>    {% include snippets/import_via_link.md %}
+>
+{: .hands_on}
+
+# Quality Control
+
+## NanoPlot to explore data
+
+
+The first thing we want to do is to get a feeling for our input data and its quality. This is done
+using the NanoPlot tool. This will create several plots, a statisical report and an HTML
+report page.
+
+![NanoPlot example](../../images/nanopore_seqeunce_analysis/NanoPlot.png)
```

Comment:
@willemdek11 can the Nanoplot galaxy tool also be made to produce this plot (of average read quality)? I was not finding the option to do so, but think it could be nice to show and then follow up this step with a filtering step by quality (or read length) orso before proceeding, what do you think?

Reply:
oh, guess we would need the fastq input, do you happen to know if the publication you got this data from had fastq files as well?

Reply:
@shiltemann At the moment it is not possible in the Nanoplot galaxy tool, because I only incorporated the plots available for all formats.